app/presenters/publishing_api_presenters/case_study.rb

```diff
@@ -1,43 +1,42 @@
 class PublishingApiPresenters::CaseStudy < PublishingApiPresenters::Edition
-  def as_json
-    super.merge(
-      format: "case_study",
-      rendering_app: edition.rendering_app,
-      links: links
-    )
-  end
-
-private
-
   def links
     {
-      lead_organisations: edition.lead_organisations.map(&:content_id),
+      lead_organisations: item.lead_organisations.map(&:content_id),
       related_policies: policy_content_ids,
-      supporting_organisations: edition.supporting_organisations.map(&:content_id),
-      document_collections: edition.published_document_collections.map(&:content_id),
-      world_locations: edition.world_locations.map(&:content_id),
-      worldwide_organisations: edition.worldwide_organisations.map(&:content_id),
-      worldwide_priorities: edition.worldwide_priorities.map(&:content_id),
+      supporting_organisations: item.supporting_organisations.map(&:content_id),
+      document_collections: item.published_document_collections.map(&:content_id),
+      world_locations: item.world_locations.map(&:content_id),
+      worldwide_organisations: item.worldwide_organisations.map(&:content_id),
+      worldwide_priorities: item.worldwide_priorities.map(&:content_id),
     }
   end
 
+private
+  def document_format
```

Comment:
Yes. While working on this I forgot to implement `format` on a class. This ended up calling `Kernel#format` which confused me for a while. I figured it was better to have a more obvious failure scenario by renaming the method and having a `NoMethodError` thrown if it wasn't implemented.